dashboard/db/migrate/20160315215912_create_paired_user_levels.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePairedUserLevels < ActiveRecord::Migration
+  def change
+    create_table :paired_user_levels do |t|
+      t.integer :driver_user_level_id
```

Comment:
This is code I inherited — I'm not sure if there are any benefits (drawbacks?) of each format.